components/02-molecules/registration/WaitingRegistrationLocktimeComponent.tsx

```diff
@@ -42,6 +43,11 @@ export const WaitingRegistrationLocktimeComponent = ({
   useEffect(() => {
     if (timer !== null && !timer) {
       setTimerDone(true);
+      if (address && nameRegistrationData.name) {
+        setNameRegistrationInLocalStorage(address, nameRegistrationData.name, {
+          timerDone: true,
+        });
```

Comment:
This logic will anyways need to be implemented in the frontEnd, whenever getting the data from a previous registration process. Whenever doing this, we will need to compare the timestamps. 

Best to not store timerDone and to do this math once we get the data AND user is in the middle of the Commit transaction / Register transaction steps

Reply:
The commit timestamp is stored in the local storage and the timer logic is being applied, the timerDone variable is used just for the purpose of checking if the user was in this step or not.